platform/darwin/test/MGLFunctionTests.m

```diff
@@ -0,0 +1,13 @@
+#import <XCTest/XCTest.h>
+#import <Mapbox/Mapbox.h>
+
+@interface MGLFunctionTests : XCTestCase
+@end
+
+@implementation MGLFunctionTests
+
+- (void)testStoplessFunction {
+    XCTAssertThrows([MGLStyleValue<NSNumber *> valueWithStops:@{}], @"Stopless function should throw an exception");
```

Comment:
If you give the exception a name, you can use `XCTAssertThrowsSpecificNamed()` here.